tests/ludwig/utils/test_defaults.py

```diff
@@ -58,6 +63,21 @@
 SCHEDULER_DICT = {"type": "async_hyperband", "time_attr": "time_total_s"}
 
 
+def _merge_preprocessing(
```

Comment:
@justinxzhao I don't think that's necessary because https://github.com/ludwig-ai/ludwig/pull/2208/files#diff-671042bc7390c5a898a22f285e44a717dc6276bc0fd3e81142e8fc468e2e6e23R35 function does the work to pass in the data just like how it was being done before, which I also think is the cleanest way to do this.

This `_merge_preprocessing` is unnecessary and I've updated the code with all that's needed for this test.